services/common/src/components/common/ScrollSidePageWrapper.tsx

```diff
@@ -8,16 +8,32 @@ interface ScrollSidePageWrapperProps {
   content: ReactNode;
   menuProps?: ScrollSideMenuProps;
   header: ReactNode;
+  extraItems?: ReactNode;
   headerHeight?: number;
+  view?: "default" | "steps" | "anchor";
 }
 
 export const coreHeaderHeight = 62; // match scss variable $header-height
 const msHeaderHeight = 80;
 
+/**
+ * A wrapper component that provides a side menu and a content area. The side menu can be fixed to the top of the page.
```

Comment:
I always really love that you add these descriptions.  I need to make more of a habit of doing that when appropriate.